__DEFINES/game.dm

```diff
@@ -52,6 +52,7 @@
 
 #define MOMMI_SOVIET_SPEED_MODIFIER 1
 #define MOMMI_NT_SPEED_MODIFIER 1
+#define COGSPIDER_SPEED_MODIFIER 2
```

Comment:
They have half of the speed of a MoMMI? :thinking:

Reply:
yup.

Clockwork is slow but sturdy

Reply:
They don't look sturdy to me, same 60 health and all.

Reply:
and they're something that's going to murder you rather than sit in the bar and pour drinks.

Reply:
yikes